src/main/webapp/js/datepicker.js

```diff
@@ -73,22 +73,6 @@ function triggerDatepickerOnClick(datepickerDivs) {
  * @assumption: startDate has a valid value
  * @returns
```

Comment:
We should remove this empty `@returns` or give more details. But that can be a separate issue. @samsontmr